---
sensor_msgs/Image depth_image
sensor_msgs/Image rgb_image
sensor_msgs/CameraInfo rgb_info
candidate_locator/ArrayImages candidates
